app/helpers/user_helper.rb

```diff
@@ -0,0 +1,21 @@
+module UserHelper
+  def line_reports_table_rows(line_reports)
+    rows = []
+
+    line_reports.each do |line_report|
+      rows << [
+        {
+          text: "#{line_report.given_name} #{line_report.family_name}",
+        },
+        {
+          text: sanitize("<strong class='govuk-tag'> PLACEHOLDER FOR STATUS </strong>"),
```

Comment:
It might be worth adding a TODO note here to ensure this isn't missed in the next step